src/ad-services/krux/index.js

```diff
@@ -82,12 +82,13 @@ class Krux {
 	 * @returns {void}
 	 */
 	importUserData() {
-		const user = getKruxData('kxuser');
+		const oldUser = getKruxData('kxuser');
+		const newUser = getKruxData('kxwikia_user');
 		const segments = getKruxData('kxsegs');
```

Comment:
Should we handle segments too?
```suggestion
		const segments = getKruxData('kxsegs') || getKruxData('kxwikia_segs');
```